### DISCUSSION TITLE
Pass a dashboard variable as uppercase ?

### DISCUSSION THREAD

**Discussion by piRGoif** [grafana/grafana-infinity-datasource#991] [Q&A]
Hello,

We've just added Infinity to our Grafana instances. The goal is to add a panel to query a partner API. 
The dashboard has a textbox variable which allows the user to set a serial number that will be used in each panels.

In the new panel, this variable is used directly in the URL : method GET, URL for example ` https://mypartner.com/api/$serial` 

With this API only the upper case serial are working. Is there any mean to add to the URL the upper case version of the textbox variable content ?

**Comment by yesoreyeram** [comment]
URL field doesn't have such functions. Instead have another hidden variable and use UQL to transform the other variable to uppercase into this new variable.

**Reply by piRGoif** [reply]
Thanks for your answer ! 

I really like the idea !
But can't manage to find how to create such variable ? 

To hide a variable, seems we only need to set the "Show on dashboard" variable option.

But I didn't find how to make the variable value computed from another variable ?
